tools/yuidoc/theme/assets/css/main.css

```diff
@@ -575,7 +575,7 @@ h2, h3, h4, h5, h6 {
 .apidocs ul.commas li:last-child:after { content: ''; }
 
 /* Keyboard shortcuts. */
-kbd .cmd { font-family: Monaco, Helvetica; }
+kbd .cmd { font-family: Monaco, Helvetica, sans-serif; }
```

Comment:
Should rather be `monospace`, right? Monaco is a monospace font

Reply:
done